Registering Pro3EM DB1 fails when wifi disabled with 'Error while extracting response...'

Is there a way I can do some debugging to work out whats going on here?

```
2024-02-01T17:45:03.528Z  INFO 1 --- [   scheduling-1] c.d.s.e.d.ShellyGen2DeviceRegistry       : Added 1, removed 0, current amount: 1 devices.
2024-02-01T17:45:03.537Z  INFO 1 --- [   scheduling-1] c.d.s.e.metrics.ShellyGen2Metrics        : Registering ShellyDevice(mac=<redacted>, address=<redacted>, name=Pro3EM DB1, type=SPEM-003CEBEU, firmwareVersion=20231219-134001/1.1.0-g34b5d4f).
2024-02-01T17:45:03.909Z  WARN 1 --- [   scheduling-1] c.d.s.exporter.client.ShellyGen2Client   : GET http://10.10.16.44/rpc/Shelly.GetStatus: HTTP Request failure: Error while extracting response for type [class click.dobel.shelly.exporter.client.api.gen2.Gen2ShellyStatus] and content type [application/json]
```

Running on the latest docker image with `SHELLY_GEN2DEVICES_HOSTS` and `SHELLY_AUTH_PASSWORD` set

Ahha  we have disabled wifi:
> Caused by: org.springframework.http.converter.HttpMessageNotReadableException: JSON parse error: Cannot construct instance of `click.dobel.shelly.exporter.client.api.gen2.WifiStatus`,

Entire json response is:
```
{
  "ble": {},
  "cloud": {
    "connected": false
  },
  "em1:0": {
    "id": 0,
    "current": 0.029,
    "voltage": 0,
    "act_power": 0,
    "aprt_power": 0,
    "pf": 0,
    "freq": 0,
    "calibration": "factory"
  },
  "em1:1": {
    "id": 1,
    "current": 0.027,
    "voltage": 0,
    "act_power": 0,
    "aprt_power": 0,
    "pf": 0,
    "freq": 0,
    "calibration": "factory"
  },
  "em1:2": {
    "id": 2,
    "current": 25.426,
    "voltage": 239.3,
    "act_power": 1543.1,
    "aprt_power": 6072.3,
    "pf": 0.57,
    "freq": 49.9,
    "calibration": "factory"
  },
  "em1data:0": {
    "id": 0,
    "total_act_energy": 0.01,
    "total_act_ret_energy": 0
  },
  "em1data:1": {
    "id": 1,
    "total_act_energy": 0.01,
    "total_act_ret_energy": 0
  },
  "em1data:2": {
    "id": 2,
    "total_act_energy": 73247.3,
    "total_act_ret_energy": 0
  },
  "eth": {
    "ip": "10.10.16.44"
  },
  "modbus": {},
  "mqtt": {
    "connected": false
  },
  "sys": {
    "mac": "34987A686808",
    "restart_required": false,
    "time": "08:46",
    "unixtime": 1707209187,
    "uptime": 399674,
    "ram_size": 241672,
    "ram_free": 130660,
    "fs_size": 524288,
    "fs_free": 184320,
    "cfg_rev": 22,
    "kvs_rev": 0,
    "schedule_rev": 3,
    "webhook_rev": 3,
    "available_updates": {
      "beta": {
        "version": "1.2.0-beta1"
      }
    },
    "reset_reason": 3
  },
  "temperature:0": {
    "id": 0,
    "tC": 33.5,
    "tF": 92.2
  },
  "wifi": {
    "sta_ip": null,
    "status": "disconnected",
    "ssid": null,
    "rssi": 0
  },
  "ws": {
    "connected": false
  }
}
```

I can provide the whole debug log, but was just searching to avoid I don't accidentally share secrets

--- a/src/main/kotlin/click/dobel/shelly/exporter/ShellyExporterApplication.kt
+++ b/src/main/kotlin/click/dobel/shelly/exporter/ShellyExporterApplication.kt
@@ -7,5 +7,6 @@ import org.springframework.boot.runApplication
 class ShellyExporterApplication
 
 fun main(args: Array<String>) {
+  @Suppress("SpreadOperator")
   runApplication<ShellyExporterApplication>(*args)
 }

--- a/src/main/kotlin/click/dobel/shelly/exporter/client/api/gen2/Gen2ShellyStatus.kt
+++ b/src/main/kotlin/click/dobel/shelly/exporter/client/api/gen2/Gen2ShellyStatus.kt
@@ -97,11 +97,11 @@ data class Temperature(
 
 data class WifiStatus(
   @JsonProperty("sta_ip")
-  val ip: String,
+  val ip: String?,
   @JsonProperty("status")
   val status: String,
   @JsonProperty("ssid")
-  val ssid: String,
+  val ssid: String?,
   @JsonProperty("rssi")
   val rssi: Int,
 )

